team_project_access.go

```diff
@@ -0,0 +1,218 @@
+package tfe
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+)
+
+// Compile-time proof of interface implementation.
+var _ TeamProjectAccesses = (*teamProjectAccesses)(nil)
+
+// TeamProjectAccesses describes all the team project access related methods that the Terraform
+// Enterprise API supports
+//
+// TFE API docs: (TODO: ADD DOCS URL)
```

Comment:
💅 probably make this look a little less incomplete. When you circle back to remove the beta tag, it would be a good idea to provide the docs URL at that time

```suggestion
TFE API docs: Documentation will be linked once this feature is available
```